app/models/checkbox_criterion.rb

```diff
@@ -177,6 +177,20 @@ def self.load_from_yml(criterion_yml)
     # a description is not given.
     criterion.description =
       criterion_yml[1]['description'].nil? ? '' : criterion_yml[1]['description']
+    # Visibility options
+    criterion.ta_visible = criterion_yml[1]['ta_visible'] unless criterion_yml[1]['ta_visible'].nil?
+    criterion.peer_visible = criterion_yml[1]['peer_visible'] unless criterion_yml[1]['peer_visible'].nil?
     criterion
   end
+
+  # Returns a hash containing the information of a single checkbox criterion.
+  def self.to_yml(criterion)
+    criterion_info = {}
```

Comment:
I tried using symbol keys, but the format changed. I will make it a single expression.